src/scorecard.rs

```diff
@@ -1,15 +1,14 @@
 use super::calchand;
-use super::hand;
-use hand::Dice;
+use crate::hand::Dice;
 use std::fmt;
 
-#[derive(PartialEq, Eq, Debug)]
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
```

Comment:
You often want simple enums like this to be `Copy`, so you don't end up having to pass them around by reference.